### DISCUSSION TITLE
Usage as a theme

### DISCUSSION THREAD

**Discussion by conformist-mw** [thuliteio/doks#1020] [General]
Hi! 

First of all, I want to thank you for such a great project!

I have already reviewed the questions and know that this project cannot be used simply as a theme for hugo (git submodule). 

I.e. an example of use is to clone the project, delete the .git directory, create your own repository (in order to gradually fill with content).

this doesn't really sound good and, the biggest question, how do I update the project in this case?

Or am I misunderstanding how to use it?

Thanks for the answers in advance!

**Comment by h-enk** [comment]
Thanks + thanks for your questions!

> [..] and, the biggest question, how do I update the project in this case?

That's the primary reason I added the Doks [child theme](https://getdoks.org/docs/overview/quick-start/#child-theme) which holds Doks as an npm module and can be updated with:

```bash
npm update @hyas/doks
```

> I.e. an example of use is to clone the project, delete the .git directory, create your own repository (in order to gradually fill with content).

Or, use a tool like [degit](https://github.com/Rich-Harris/degit) to scaffold your project:

```
npx degit h-enk/doks-child-theme my-doks-project
```

Or, use this button on [h-enk/doks-child-theme](https://github.com/h-enk/doks-child-theme):

![Snag_1a8432ee](https://user-images.githubusercontent.com/3902872/223781322-78a7e1d6-3811-427a-a8f7-c0806dbd15d7.png)

> this doesn't really sound good [..]

Totally agree. Doks 1.0 (in the making) is restructured fully in several (logical) npm modules (making it even more easy to update). With Doks 1.0 the child theme will become obsolete. Also, Doks 1.0 will use a scaffold project which you can set up with the new Hyas CLI (available shortly — Doks is a Hyas theme). See also [Start your first project](https://deploy-preview-256--gethyas.netlify.app/docs/start-here/getting-started/#start-your-first-project)

**Comment by conformist-mw** [comment]
Thank you very much for the answers, everything became clear. I will try to use it as a child and will wait for version 1.0.
Cheers!
